Update conflict constraint for "nelmio/api-doc-bundle" in order to allow version 3 (#478)
sonata-project/SonataNotificationBundle: UPGRADE-3.x.md

UPGRADE FROM 3.x to 3.x
=======================

> 

Reviewer: ```suggestion
```

There is a minimal BC Break on `MessageManager::getPager`. If you are extending this method you should add parameter and return type hints.

### Support for NelmioApiDocBundle > 3.6 is added

> Controllers for NelmioApiDocBundle v2 were moved under `Sonata\NotificationBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.

Reviewer: ```suggestion
Controllers for NelmioApiDocBundle v2 were moved under `Sonata\NotificationBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you MUST ensure they are using the corresponding inheritance.
```

Author: Done.

> Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\NotificationBundle\Controller\Api\Legacy` namespece and controllers for NelmioApiDocBundle v3 was added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.

Reviewer: ```suggestion
Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\NotificationBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
```
